src/main/java/Duke.java

```diff
@@ -1,10 +1,124 @@
+import java.util.Scanner;
+
 public class Duke {
+    // constant for number of maximum tasks
+    public static final int MAXIMUM_TASKS = 100;
```

Comment:
Good job for declaring constant in all uppercase which conforms to coding standard, however, perhaps you can change it to `private static` instead since it's only being used in this class, this would improve your encapsulation.